<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectInfoWidget</class>
 <widget class="QDialog" name="ProjectInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Project Info</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Title:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="titleLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="gameLabel">
     <property name="text">
      <string>Game/Album:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="gameLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="authorLabel">
     <property name="text">
      <string>Author:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="authorLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="releaseDateLabel">
     <property name="text">
      <string>Release Date:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDateEdit" name="releaseDateEdit">
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="date">
      <date>
       <year>2024</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="notesLabel">
     <property name="text">
      <string>Notes:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" rowspan="2">
    <widget class="QPlainTextEdit" name="notesTextEdit"/>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>204</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="showOnOpenCheckBox">
     <property name="text">
      <string>Show info when project is opened</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectInfoWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectInfoWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
